### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by AlyAtalla**
Used jest framework for testing.
Created a test file ([..].test.js) for a file containing the add item and delete item functions to test.
Mock HTML to test if add/delete functions add or remove exactly one <li> element to/from the list in the DOM.
grouped  tests using the describe() method.

### CODE REVIEW

File: src/modules/add-delete.test.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      store[key] = value.toString();
+    },
+    removeItem: (key) => {
+      delete store[key];
+    },
+    clear: () => {
+      store = {};
+    },
+  };
+})();
```
Comment (iambenkis) [inline src/modules/add-delete.test.js]: - [x] Well done making most of the tests passed 👏  , however there are still some you still need to fix . I know they may be working on your computer but always remember that most of the product you will make will be running on clients computers 🆗  , Kindly fix the unpassed tests to meet the clients need 😄 😄 😄 
- [x] Also I noticed that there is no snapshot test running , kindly fix that also 👍 

Capture

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi Team 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  Unit tests are passing 

## Suggested changes

- [x] Kindly provide information about tests , how to install them , how to run them as they are important part of you project. The usage sub-section should be present in that case 👍 

Capture


_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/modules/add-delete.test.js (hunk 1 above)
Comment (AlyAtalla) [reply src/modules/add-delete.test.js]: thank you

**Review by christianbiring1 — APPROVED** [review]
Hi @AlyAtalla , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Great work for implementing the changes as required in the previous review :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**